docs/user/reference/metrics/event.md

```diff
@@ -423,7 +377,10 @@ use metrics::views;
 
 assert_eq!(
     0,
-    views::login_opened.test_get_num_recorded_errors(ErrorType::InvalidOverflow, None)
+    views::login_opened.test_get_num_recorded_errors(
+        ErrorType::InvalidOverflow,
+        None
```

Comment:
Huh, here too with the second argument. Though this one seems to have come from the previous version of the doc.